src/main/java/org/bunge/bgginfo/bggintegration/service/DefaultBGGService.java

```diff
@@ -1,33 +1,61 @@
 package org.bunge.bgginfo.bggintegration.service;
 
-import java.util.List;
-
 import org.bunge.bgginfo.bggintegration.model.BGGPayloadSearchResults;
 import org.bunge.bgginfo.game.GameDAO;
 import org.bunge.bgginfo.game.GameDTO;
 import org.bunge.bgginfo.game.GameRepo;
 import org.bunge.bgginfo.playerspoll.PlayersPollRepo;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 import org.springframework.web.reactive.function.client.WebClient;
 
+import javax.annotation.PostConstruct;
+import java.util.List;
+
 @Service
 public class DefaultBGGService implements BGGService {
+    private static final Logger log = LoggerFactory.getLogger(DefaultBGGService.class);
+    private static final String BGG_URL_TEMPLATE = "https://www.boardgamegeek.com/xmlapi2";
+    private static final int BGG_BATCH_SIZE = 20;
+    private static final int BGG_MAX_GAME_ID = 5000000;
 
```

Comment:
So something that I might not have explained well is the point of the BGG Service is to run perpetually, so that the any updates that need to be made to the data can be done.

I'll leave it up to you if you want make that change or not because as it stands this PR is a significant improvement over the original project in multiple aspects so I have no qualms approving it as is.